app/views/plans/edit.html.haml

```diff
@@ -3,7 +3,7 @@
 
 #editform
   =form_for @plan, :url => {:action=>:update, :controller=>:plans } do |f|
-    =f.text_area :edit_text, :onkeyup=>"javascript:checkPlanLength()"
+    =f.text_area :edit_text, :rows=>"14", :onkeyup=>"javascript:checkPlanLength()"
```

Comment:
Yeah, sort of, I was just lining the bottom of the edit box up with the search bar on the sidebar.